plugins/janus_videoroom.c

```diff
@@ -987,11 +1010,15 @@ void janus_videoroom_destroy_session(janus_plugin_session *handle, int *error) {
 		if(session->participant_type == janus_videoroom_p_type_publisher) {
 			/* Get rid of publisher */
 			janus_videoroom_publisher *p = (janus_videoroom_publisher *)session->participant;
```

Comment:
There is no need to increase the refcount for the publisher here as it is gettting cleared.  However, the session mutex locked one line later should protect this assignment, too.